spec/tom_queue/tom_queue_integration_spec.rb

```diff
@@ -253,7 +253,7 @@ def start!
 
       # Now merge all the consumers internal work arrays into one
       # sorted by the received_at timestamps
-      sink_order = consumers.map { |c| c.work }.flatten.sort.map { |a| a.payload }
+      sink_order = consumers.map(&:work).flatten.sort { |a, b| a[:run_at] <=> b[:run_at] }.map(&:payload)
```

Comment:
See above, breaks the tests in such a way that they reliably pass, but don't actually test anything ;)